src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Regex.cs

```diff
@@ -134,11 +134,14 @@ private void Init(string pattern, RegexOptions options, TimeSpan matchTimeout, C
         /// </summary>
         private static SRM.Regex InitializeSRM(RegexNode rootNode, RegexOptions options)
         {
-            // TBD: this could potentially be supported quite easily
+            // TBD: this could potentially be supported quite easily but is not of priority
```

Comment:
Yeah, we also excluded RTL from a bunch of the optimizations we did in .NET 5, as it's such a rarely used option it wasn't worth it.